ccd-definition/FixedLists/CareSupervision/AuthorityFixedList.json

```diff
@@ -2,115 +2,134 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "AuthorityFixedList",
-    "ListElementCode": "SA",
-    "ListElement": "Swansea City Council"
+    "ListElementCode": "CDF",
+    "ListElement": "Cardiff Council",
+    "DisplayOrder": 5
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "AuthorityFixedList",
-    "ListElementCode": "HN",
-    "ListElement": "London Borough Hillingdon"
+    "ListElementCode": "ERY",
+    "ListElement": "East Riding of Yorkshire Council",
+    "DisplayOrder": 10
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "AuthorityFixedList",
-    "ListElementCode": "STF",
-    "ListElement": "Staffordshire County Council"
+    "ListElementCode": "FPLA",
```

Comment:
I know that it was already in there but should this be here, does it describe us or is it for something else? If it is us would it not make more sense for it to be in a -nonprod file.

Reply:
@lewisbirks , confirmed with Adam, this is legacy org used long time ago to do smoke testing in prod. Currently not used. Cannot submit case for such org, see fpl:
  local_authority_codes_forbidden_case_submission: FPLA